test/fulfillmentSpec.js

```diff
@@ -177,6 +178,57 @@ describe('GET /fulfillment', function () {
       .expect(this.executionConditionFulfillment)
       .expect(validator.validateFulfillment)
       .end()
+  }
+
+  it('should return a fulfillment', returnFulfillment)
+
+  describe('when hold account is busy', function () {
+    const ARGS_EXPECTED = {
+      '0': 'UPDATE "L_ACCOUNTS" SET "BALANCE" = "BALANCE" + ? WHERE "NAME" = ?',
+      '1': [ -10, 'hold' ]
+    }
+
+    beforeEach(function () {
+      dbFailureMock.timesQueryShouldFail[JSON.stringify(ARGS_EXPECTED)] = 3
+    })
+
+    afterEach(function () {
+      const timesLeft = dbFailureMock.timesQueryShouldFail[JSON.stringify(ARGS_EXPECTED)]
+      dbFailureMock.timesQueryShouldFail[JSON.stringify(ARGS_EXPECTED)] = 0
+      expect(timesLeft).to.equal(0)
+    })
+
+    it('should return a fulfillment', returnFulfillment)
+
+    it('should give up if retrying takes too long', function * () {
+      dbFailureMock.timesQueryShouldFail[JSON.stringify(ARGS_EXPECTED)] = 1000000
+      try {
+        const transfer = this.preparedTransfer
+
+        yield dbHelper.setHoldBalance(10)
+
+        yield this.request()
+          .put(transfer.id + '/fulfillment')
+          .auth('admin', 'admin')
+          .send(this.executionConditionFulfillment)
+          .expect(500)
```

Comment:
done. I added the 503 responses in all API points for PUT, not for the GET ones.
Do you also want me to create unit tests that check the 503 responses for the other API end points, or is it ok like this?